src/i18n/en/translation.json

```diff
@@ -19,6 +19,9 @@
     "pleaseProvideAdditionalText": "Please provide additional text",
     "pleaseListenToAudio": "Please listen to the audio until the end.",
     "onlyNumbersAllowed": "Only numbers are allowed",
+    "answerTooLarge": "Please provide a shorter answer",
+    "question_count_plural": "{{length}} questions",
+    "question_count_singular": "{{length}} question",
```

Comment:
I didn't find these question_count_singular, question_count_plural on fr translation, is it right ?

Reply:
oh I see, so question_count_plural and question_count_singular is not relevant to be added on fr Translation ?
I understand its a point that you only resolved the conflict, but I didnt find this on fr Translation anyway, if it is not relevant ok, we can proceed without add those variable to fr translation.

Reply:
no haha, I suggested to add on fr Translation, but I see, ok if you dont think its relevant, for me that's ok, I agree with Carlos review and you can resolve my review.

Reply:
Good catch @felipeMetaLab 

@ramirlm this is a side effect from your merge from dev into this branch. These lines were removed in [this PR](https://github.com/ChildMindInstitute/mindlogger-web-refactor/pull/530/files#diff-748406503f0dd61aa7860bd36e02a6674aac6247b7e9bbf07dd262966fbaeff0) so they shouldn't be added back in here.